db/migrate/20171129130243_create_items.rb

```diff
@@ -5,6 +5,7 @@ def change
       t.references :user, foreign_key: true, type: :uuid
       t.references :transaction, foreign_key: true, type: :uuid
       t.references :article, foreign_key: true, type: :uuid
+      t.references :amount, foreign_key: true, :default => 1
```

Comment:
This shouldn't be a t.references, it should be a t.integer (Its the type of the column )